quickstart.py

```diff
@@ -0,0 +1,40 @@
+import great_expectations as gx
+
+
+# Set up
+context = gx.get_context()
+
+
+# Connect to data
+validator = context.sources.pandas_default.read_csv(
+    "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv",
+    asset_name="taxi_asset",
```

Comment:
Need to specify an asset name to get this to persist

Reply:
At the beginning of last week we talked about auto-generating an asset name when doing this for cloud and making this change to the quickstart unnecessary. 
Did we decide against that? Or is this just an intermediate step to get the quick-start working while we make the more substantive changes?

cc @DrewHoo

Reply:
Ah yeah I considered that a follow-up step (but thanks for calling it out!). I'm just trying to get a barebones version working so we can iterate on it in the epic.

Reply:
Also, we could temporaliy make this quickstart prevent asset collisions here by making `asset_name=f"taxi_asset-{uuid.uuid4()}", or something similar.